powershell.go

```diff
@@ -22,6 +23,17 @@ func Powershell(psCmd string) string {
 	// Finally make it base64 encoded which is required for powershell.
 	psCmd = base64.StdEncoding.EncodeToString([]byte(encoded))
 
-	// Specify powershell.exe to run encoded command
-	return "powershell.exe -EncodedCommand " + psCmd
+	var process string
+
+	// Windows
+	if runtime.GOOS == "windows" {
+		// Specify powershell.exe to run encoded command
+		process = "powershell.exe"
+	}
+	// Linux // MacOS
+	if runtime.GOOS == "linux" || runtime.GOOS == "darwin" {
```

Comment:
I'm not sure I get the rationale here. The executable to launch is dependent on the target platform not the platform where the winrm client is running, or did I miss something ?